hphp/vixl/a64/constants-a64.h

```diff
@@ -1465,11 +1465,11 @@ enum LSELoadOp {
   LSE_LDUMIN = 0x00007000
 };
 
-enum LSEAquireRelease {
-  LSEAquireReleaseN  = 0x00000000,
-  LSEAquireReleaseA  = 0x00800000,
-  LSEAquireReleaseL  = 0x00400000,
-  LSEAquireReleaseAL = 0x00C00000
+enum LSEAcquireRelease {
+  LSEAcquireReleaseN  = 0x00000000,
+  LSEAcquireReleaseA  = 0x00800000,
+  LSEAcquireReleaseL  = 0x00400000,
+  LSEAcquireReleaseAL = 0x00C00000
```

Comment:
The changes here are by far the scariest. I'm happy to drop them if they're a problem.

Reply:
I think they are copied from https://github.com/armvixl/vixl. Maybe we should not fix typo in this library.

Reply:
It looks like that library moved here: https://github.com/Linaro/vixl, I've added it to my list.